Nef_3/include/CGAL/Nef_polyhedron_3.h

```diff
@@ -1710,6 +1710,8 @@ class Nef_polyhedron_3 : public CGAL::Handle_for< Nef_polyhedron_3_rep<Kernel_,
     bool ninety = is_90degree_rotation(aff);
     bool scale = is_scaling(aff);
 
+    bool translate = aff.is_translation();
```

Comment:
I like this approach, and wonder if `is_scaling` might also find itself in the Aff_transformation_3 classes oneday?

Reply:
That's what the _Todo_ is about.